prowler/providers/aws/services/elasticache/elasticache_redis_cluster_auth_enabled/elasticache_redis_cluster_auth_enabled.py

```diff
@@ -0,0 +1,32 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.elasticache.elasticache_client import (
+    elasticache_client,
+)
+
+
+class elasticache_redis_cluster_auth_enabled(Check):
+    def execute(self):
+        findings = []
+        for repl_group in elasticache_client.replication_groups.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = repl_group.region
+            report.resource_id = repl_group.id
+            report.resource_arn = repl_group.arn
+            report.resource_tags = repl_group.tags
+
+            cluster = repl_group.member_clusters[0]
```

Comment:
Is there no other way to get the version of the replication cluster? It's weird to get it from the first cluster, it's a bit arbitrary.

Reply:
I´ll update this in a better way. I thought `MemberClusters` returned primary node in first position, but I think that´s incorrect. I´ll take the primary node now (whose version is replicated in replica nodes) to make this non arbitrary at all.